VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -352,22 +351,21 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
         }, this.audioStreamIntervalSeconds * 1000);
     }
 
-    continueWithNoRecordingCallback() {
-        if (this.audioErrorRetryToast.actioned) {
-            this.continueWithNoRecording = true;
-        }
-        this.audioErrorToastOpen = false;
+    audioRestartCallback(continueWithNoRecording: boolean) {
+        this.continueWithNoRecording = continueWithNoRecording;
+        this.audioErrorRetryToast = null;
     }
 
     async retrieveAudioStreamInfo(hearingId): Promise<void> {
         if (this.conference.status === ConferenceStatus.InSession) {
+            this.logger.debug(`${this.loggerPrefixJudge} Recording Session Seconds: ${this.recordingSessionSeconds}`);
             this.recordingSessionSeconds += this.audioStreamIntervalSeconds;
         } else {
             this.recordingSessionSeconds = 0;
             this.continueWithNoRecording = false;
         }
 
-        if (this.recordingSessionSeconds > 20 && !this.continueWithNoRecording) {
+        if (this.recordingSessionSeconds > 30 && !this.continueWithNoRecording && this.showVideo && !this.audioErrorRetryToast) {
```

Comment:
Not a deal-breaker but the number 30 here is a bit arbitrary. A comment or a variable with a name for that the 30 represents would be helpful here

Reply:
Cheers, will update it in my next pr